static/_redirects

```diff
@@ -105,3 +105,4 @@
 /docs/pipeline-components-and-applications/version-compatibility-matrix/* /docs/pipeline-components-and-applications/versions/ 301
 
 /docs/tutorials/tutorial-a-guide-to-using-the-poplin-chrome-extension/* /docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/:splat 301
+/docs/destinations/forwarding-events/snowbridge/configuration/transformations/testing/* /docs/destinations/forwarding-events/snowbridge/testing/:splat 301
```

Comment:
Presumably this just redirects anyone from old links to new page? Nice.

Reply:
Yeah, actually the `*` and `:splat` are unnecessary in this case, will remove